### PR TITLE
Update to debian stretch now that jessie is moving to LTS

### PR DESCRIPTION
**Pull request by yosifkit**

### CODE REVIEW

**Comment by jessfraz** [comment]
LGTM

File: debian/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM debian:jessie
+FROM debian:stretch-slim
```
Comment (tianon) [inline debian/Dockerfile:1]: End of an era, Jess. :disappointed: :cry:

Comment (jessfraz) [reply debian/Dockerfile:1]: :broken heart:

Comment (qubbit) [reply debian/Dockerfile:1]: 💔

**Pull request merged**